File: src/main/java/com/asap/server/controller/ServerProfileController.java
package com.asap.server.controller;

import io.swagger.v3.oas.annotations.Hidden;

import java.util.Arrays;

import lombok.RequiredArgsConstructor;
import org.springframework.core.env.Environment;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequiredArgsConstructor
@Hidden
public class ServerProfileController {

    private final Environment env;

    @GetMapping("/profile")
    public String getProfile() {
        return Arrays.stream(env.getActiveProfiles())
                .findFirst()
                .orElse("");
    }
}

File: presentation/src/main/kotlin/com/asap/server/controller/ServerProfileController.kt
package com.asap.server.controller

import org.springframework.core.env.Environment
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController
import java.util.*

@RestController
class ServerProfileController(
        private val env: Environment
) {
    @GetMapping("/profile")
    fun getProfile(): String {
        return Arrays.stream(env.activeProfiles)
                .findFirst()
                .orElse("")
    }
}